src/Editor/EditorSupport.php

```diff
@@ -51,7 +50,12 @@ public function maybe_show_notice() {
 			return;
 		}
 
-		if ( ! isset( $screen->is_block_editor ) || false === $screen->is_block_editor ) {
+		if (
+			! empty( $screen->is_block_editor )
+			||
+			// Applicable to Gutenberg v5.5.0 and older.
+			( function_exists( 'is_gutenberg_page' ) && ! is_gutenberg_page() )
+		) {
```

Comment:
Is this not inverted from what it was before? The `! empty( $screen->is_block_editor )` condition means it _is_ the block editor, and we should only be returning here if it is not the block editor, right?

```suggestion
		$is_block_editor = (
			! empty( $screen->is_block_editor )
			||
			// Applicable to Gutenberg v5.5.0 and older.
			( function_exists( 'is_gutenberg_page' ) && ! is_gutenberg_page() )
		);
		if ( ! $is_block_editor ) {
```

Reply:
Otherwise it seems the notice would never be displayed when Gutenberg 5.6+ is active.

Reply:
Whoops yea you're right.

Reply:
Indeed, when testing with WordPress 5.1.3 and Gutenberg 5.6.1 I see no notice.